### DISCUSSION TITLE
How can you manage the size of cell results in jupyter notebook so that beamer will look good?

### DISCUSSION THREAD

**Discussion by Mgonrub** [quarto-dev/quarto-cli#5898] [Q&A]
### Description

I am trying to show the output of this python code in jupyter notebook.
`xxx1=df_fil2.groupby('Inte')['2023_Costs'].describe().round(1)
xxx1`
But the result is out of the beamer slide, it's a table but the same thing happens to me when I deal with an image as a graphic.
![image1](https://github.com/quarto-dev/quarto-cli/assets/130856876/4ecf1b74-476f-4eb8-81b1-219bb0e4ff7b)

**Comment by mcanouil** [comment]
Could you share a small self-contained "working" (reproducible) example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````

**Comment by Mgonrub** [comment]
Or how is this done in jupyter notebook?

Highlighting important sentences/words

`\frametitle{Sample frame title}

In this slide, some important text will be
\alert{highlighted} because it's important.
Please, don't abuse it.

\begin{block}{Remark}
Sample text
\end{block}

\begin{alertblock}{Important theorem}
Sample text in red box
\end{alertblock}

\begin{examples}
Sample text in green box. The title of the block is ``Examples".
\end{examples}`

Or

`\begin{stepenumeratewithalert}
\item La Geo".

\item Ell.

\item Fue.

\item Ere.

\item Apl.

\item Analis
\end{stepenumeratewithalert}`

**Reply by mcanouil** [reply]
Please follow the example I provided, copy/paste it and put your code inside.
We need actual Quarto document to provide effective help.
You are only providing LaTeX code.

For LaTeX questions you might want to target more specialised places such as StackExchange or StackOverflow.

Note that Pandoc/Quarto accepts raw LaTeX code, see an example below.

````
```{=latex}
\begin{stepenumeratewithalert}
\item La Geo".

\item Ell.

\item Fue.

\item Ere.

\item Apl.

\item Analis
\end{stepenumeratewithalert}
```
````

**Reply by mcanouil** [reply]
See "callouts" in the documentation (<https://quarto.org/docs/authoring/callouts.html>).

**Comment by mcanouil** [comment]
Regarding, the size of your Python table, see #5855 for similar "issue", thus similar answer.
